Remote work newsletter (#8154)
PostHog/posthog.com: contents/newsletter/remote-working.md

**This week, we explain: **how to make remote work actually work.

> This article was first published in our newsletter, [Product for Engineers](https://newsletter.posthog.com/). It's all about helping engineers and founders build better products by learning product skills. [Subscribe here](https://newsletter.posthog.com/subscribe).

## The problem? You’ve got a two-tiered system

If you’re a company built around [big](https://www.cnbc.com/2023/06/08/google-to-crack-down-on-hybrid-work-asks-remote-workers-to-reconsider.html), [shiny](https://gizmodo.com/apple-remote-work-return-to-office-hybrid-work-1850256210) [offices](https://www.cnn.com/2022/06/01/tech/elon-musk-tesla-ends-work-from-home/index.html), then chances are you have a two-tiered system. Essentially, an employee’s ability to move up in such a company is closely linked to their proximity to the biggest, shiniest office. Dell’s new policy acknowledges this outright: if you’re a remote worker at Dell, there’s no career progression. The same is true at many companies that have a mix of remote, hybrid, and office workers.

### The solution? Aggressive transparency

> Rather than presentism and nepotism, career progression should be linked to a transparent framework of measurable goals. Do the work, do it well, deliver value for customers, and you’ll be rewarded. Leif Singer, a software engineer at Automattic, the company behind WordPress, has shared some great examples of [how this works in practice](https://youtu.be/aRiP72kRrnc?feature=shared). 

Reviewer: +1

> Rather than presentism and nepotism, career progression should be linked to a transparent framework of measurable goals. Do the work, do it well, deliver value for customers, and you’ll be rewarded. Leif Singer, a software engineer at Automattic, the company behind WordPress, has shared some great examples of [how this works in practice](https://youtu.be/aRiP72kRrnc?feature=shared). 

Author: ```suggestion
Rather than presentism and nepotism, career progression should be linked to a transparent framework of measurable goals. Do the work, do it well, deliver value for customers, and you’ll be rewarded. Leif Singer, a software engineer at Automattic, the company behind WordPress, has shared some great examples of [how this works in practice](https://youtu.be/aRiP72kRrnc?feature=shared). Key points: 

- Give employees the freedom grow in ways they find interesting
- Review compensation regularly based on transparent, objective metrics
- Ensure that employees both get and give regular feedback on their work and career progression```

> Rather than presentism and nepotism, career progression should be linked to a transparent framework of measurable goals. Do the work, do it well, deliver value for customers, and you’ll be rewarded. Leif Singer, a software engineer at Automattic, the company behind WordPress, has shared some great examples of [how this works in practice](https://youtu.be/aRiP72kRrnc?feature=shared). 

Author: updated above with some more detail

A two-tiered system is built on the bizarre notion that presenteeism is how you get ahead. Clearly defining and then publishing your system for career progression is an important first step to making sure that everyone, regardless of location, is assessed using the same objective criteria. At PostHog, the way we approach [career progression](https://posthog.com/handbook/people/career-progression) and [feedback](https://posthog.com/handbook/people/feedback) is laid out in public for anyone to see.

> But transparency doesn’t stop there. Remote work pioneer GitLab does everything out in the open and has published a [2,000-page handbook](https://handbook.gitlab.com/handbook/values/) covering, well, everything. At PostHog, [our handbook](https://posthog.com/handbook) is one of the most important documents we’ve ever written. Why? If you want to find out anything about the company, you can – from what [individual teams](https://posthog.com/teams) are working on right now, to how much we would pay to [hire a full stack engineer in Brunei](https://posthog.com/handbook/people/compensation), to how our co-founders [like](https://posthog.com/handbook/company/team/james-hawkins) to [work](https://posthog.com/handbook/company/team/tim-glaser). This transparent approach gives everyone access to the same information and allows them to focus on doing their best work, not navigating layers of corporate bureaucracy.

Author: Wary about adding another bullet list here after adding one a couple of pars earlier. See what you think once that change has made

### The problem? Communication is messy

> Making remote work actually work requires making big cultural and procedural changes. Fixing how your company communicates is one of the hardest as it requires everyone to buy in and change their behavior. That means saying goodbye to email, setting fire to Microsoft Teams, and training people to be much more effective communicators. Here’s how.

Reviewer 2: Just generally, I think we need change the language to more about "building" a successful remote company, rather than fixing an existing one. Our audience is founders / aspiring founders building their own companies, not existing execs trying to fix their bad systems.

TL;DR: It's "avoid these mistakes" not "how to fix your mistakes"

Author: Can you give a few specific examples so I can zero in on where the language is wrong? Thanks!

Reviewer 2: Some examples:

> "If you’re a company built around [big](https://www.cnbc.com/2023/06/08/google-to-crack-down-on-hybrid-work-asks-remote-workers-to-reconsider.html), [shiny](https://gizmodo.com/apple-remote-work-return-to-office-hybrid-work-1850256210) [offices](https://www.cnn.com/2022/06/01/tech/elon-musk-tesla-ends-work-from-home/index.html), then chances are you have a two-tiered system."

Rather than saying "If you're a company..." maybe replace with just "Companies built around..."

> "But the policy actually makes sense for Dell. Sort of. When you’re a decades-old behemoth built around people working in offices, implementing remote work policies that actually work is hard. But it doesn't have to be. There are some key principles that any company – big or small, young or old, or Dell – can follow to realize the potential of remote working."

Reframe to make it more about "here's how to avoid becoming Dell" / "you don't have to build you company like this" etc. Might be cool to mention massive companies that have done remote/hybrid successfully. MongoDB is a famous one afaik.

Author: have hopefully addressed this in minor edits throughout

### The solution: Write. Everything. Down

> How do I [file expenses](https://posthog.com/handbook/people/spending-money)? What’s the [onboarding process for new hires](https://posthog.com/handbook/people/onboarding)? What’s the product analytics team [focused on this quarter](https://posthog.com/teams/product-analytics)? In most companies, answers to these kinds of questions are either hidden in labyrinthian collections of out-of-date Google Docs or inside someone’s brain. At well-functioning, all-remote companies, they’re written down in a handbook where anyone can access them. That handbook can be your public website, or an internal wiki. But written communication also includes Slack, GitHub, or systems like Trello or Airtable.

Reviewer 2: ```suggestion
How do I [file expenses](https://posthog.com/handbook/people/spending-money)? What’s the [onboarding process for new hires](https://posthog.com/handbook/people/onboarding)? What’s the product analytics team [focused on this quarter](https://posthog.com/teams/product-analytics)? 

In most companies, answers to these kinds of questions are either hidden out-of-date Google Docs or inside someone’s brain. At well-functioning, all-remote companies, they’re written down in a handbook where anyone can access them. That handbook can be your public website, or an internal wiki. But written communication also includes Slack, GitHub, or systems like Trello or Airtable.
```

> Defaulting to written communication means, chances are, if someone has a question they don’t have to wait for an answer – they can go find it themselves. And if they can’t find an answer, writing a clear, concise question in a public channel on Slack will ensure that everyone can see the answer. Being able to clearly communicate often complex ideas is a key skill for anyone working remotely, but especially engineers. Software engineer Dmitry Kudryavtsev [sums it up nicely](https://www.yieldcode.blog/post/why-engineers-should-write/): “The sooner you get better at writing, the faster you will become a better engineer.” It also created some lively discussion on [Hacker News](https://news.ycombinator.com/item?id=36659166).

Reviewer 2: ```suggestion
Defaulting to written communication means, chances are, if someone has a question they don’t have to wait for an answer – they can go find it themselves. And if they can’t find an answer, writing a clear, concise question in a public channel on Slack will ensure that everyone can see the answer. 

Being able to clearly communicate often complex ideas is a key skill for anyone working remotely, but especially engineers. Software engineer Dmitry Kudryavtsev [sums it up nicely](https://www.yieldcode.blog/post/why-engineers-should-write/): “The sooner you get better at writing, the faster you will become a better engineer.” It also created some lively discussion on [Hacker News](https://news.ycombinator.com/item?id=36659166).
```

> Automattic’s Beau Lebens has written (of course) a [great guide](https://beau.blog/2020/03/remote-work-at-scale/) to the benefits of effective written communication with lots of practical tips. One key takeaway: tools are not the answer. Installing a bunch of bots to transcribe meetings and run automated functions in Slack will only work if you do all the hard work first. And nailing how you communicate is a big part of that. Writing things down is great because:

Reviewer 2: As per earlier comment, would prefer to have more than one takeaway here, probably bullet point them. Can replace / combine if the following section if that makes sense.

Author: Sorry, don't quite follow. Do you mean replacing/reworking the bullets on written communication with bullets that cover both written communication and other points/takeaways? If so, what do you reckon those could be?

* **It’s faster. **Communicating via writing, especially with remote staff in lots of different time-zones, saves a lot of time. Scheduling is hard and meetings interrupt deep work. If something can be written down, it will both save time and improve transparency around decision-making.

* **It’s always there. **If a decision is made on a call and nobody writes it down, was a decision even made? Calls can be good, but calls are also information black holes. Transparency is crucial in remote-only companies. So, when you do have a meeting about something, make sure someone, [or something](https://www.sembly.ai/), is taking notes.

* **It’s clearer. **Seriously, have you ever read a transcript of you speaking? Writing things down helps with intentionality as, hopefully, it makes you think through both what you’re trying to achieve and how you want to communicate it.

> ### The problem: Meetings

Reviewer 2: ```suggestion
## Problem #3: Meetings
```

What about meetings, exactly? Too many of them, unproductive ones?

Online meetings are the scourge of remote work – and often a cause of tension between office-based and non-office based employees. Mics off when they should be on, screenshare chaos, [Jeffrey Toobin’s penis](https://www.wired.com/story/jeffrey-toobin-zoom-penis/) – they remain a technical nightmare and a productivity black hole. Many companies see them as a necessary evil – a way for colleagues, no matter where they’re based, to collaborate. This is wrong.

> Since the pandemic, the amount of time people have spent not just in online meetings, but in meetings in general, has soared. [Time spent in meetings](https://www.microsoft.com/en-us/worklab/work-trend-index/great-expectations-making-hybrid-work-work) has gone up 253%. Some Teams users [spend](https://www.microsoft.com/en-us/worklab/work-trend-index/will-ai-fix-work) close to eight hours a week, or an entire workday, in online meetings. Eight hours a week! In Teams! Madness. And if you’re a manager or executive within an organization, this is all your fault.

Reviewer 2: Would be good if we could combine these two paras. The stats are fun, but I started to lose interest there.

Author: ```suggestion
Online meetings are the scourge of remote work – and often a cause of tension between office-based and non-office based employees. Mics off when they should be on, screenshare chaos, [Jeffrey Toobin’s penis](https://www.wired.com/story/jeffrey-toobin-zoom-penis/) – they remain a technical nightmare and a productivity black hole. Many companies see them as a necessary evil – a way for colleagues, no matter where they’re based, to collaborate. This is wrong.

Since the pandemic, [time spent in online meetings](https://www.microsoft.com/en-us/worklab/work-trend-index/great-expectations-making-hybrid-work-work) has gone up 253%. Some Teams users [spend](https://www.microsoft.com/en-us/worklab/work-trend-index/will-ai-fix-work) close to eight hours a week, or an entire workday, in online meetings. And if you’re a manager or executive within an organization, this is all your fault.
```

Author: Trimmed down. Can rework to cut to one par if that's better

Reviewer 2: Cool. I still feel like one par might better, but don't feel super strongly. We can park this for now and see how we feel after after other changes flow in.

Way back in 2009, Y Combinator co-founder Paul Graham [wrote](https://www.paulgraham.com/makersschedule.html) about the difference between the “maker’s schedule” and the “manager’s schedule.” In an office setting, these two schedules (sort of) work side by side. Managers spend their days in hour-long chunks full of meetings, check-ins, brainstorms, and walking around open-plan offices. A maker’s schedule, which is basically the schedule of an engineer, is the opposite: deep work, minimal meetings, headphones, Slack. When the pandemic forced almost every office-based company in the world to go full-time remote, these two worlds smashed together.

At a lot of companies, this has forced way too many people onto the “manager’s schedule.” If this is your company, burn your weekly schedule to the ground and start again. At PostHog, we’re [anti-meeting by default](https://posthog.com/blog/meetings). We have two meeting-free days a week and, when we do have meetings, they’re short and focused. This allows not just engineers, but everyone at the company, to focus on doing their best work how they see best. We communicate asynchronously via GitHub and Slack. 

> A lot of companies, big and small, have adopted this approach. GitLab and Automattic for starters, but also [Twist](https://async.twist.com/asynchronous-communication/), [Buffer](https://buffer.com/about), and [Zapier](https://zapier.com/jobs/culture-and-values-at-zapier) have all followed a similar blueprint to make remote work actually work for both managers and engineers.

Reviewer 2: This section doesn't totally work for me atm:

- "Kill meetings" feels a bit too glib / obvious a solution. Could be more like "Async communication by default" but better than that. 😅

- Our readers will understand the Marker's schedule / managers schedule already, so no need to explain it too much.

- Would be better if this section just explained in bullet points how various successful remote companies (including us) deal with this problem. Basically, take the examples in the last para and expand on each one individually. Hopefully each does something unique people can take inspiration from.

Author: ```suggestion
### The solution: Kill meetings

Way back in 2009, Y Combinator co-founder Paul Graham [wrote](https://www.paulgraham.com/makersschedule.html) about the difference between the “maker’s schedule” and the “manager’s schedule.” In an office setting, these two schedules (sort of) work side by side. But when the pandemic forced almost every office-based company in the world to go full-time remote, things got messy.

At a lot of companies, this has forced way too many people onto the “manager’s schedule.” If this is your company, burn your weekly schedule to the ground and start again. Then rebuild it and follow these remote work commandments:

**Meeting-free days.** At PostHog, we’re [anti-meeting by default](https://posthog.com/blog/meetings). We have two meeting-free days a week and, when we do have meetings, they’re short and focused. 
**Communicate asynchronously.** We communicate asynchronously via GitHub and Slack. There are lots and tools and systems for this, but the success of asynchronous communication really comes down to good, clear writing.
**Focus on deep work.** Fewer meetings means more time for deep work. This allows not just engineers, but everyone at the company, to focus on doing their best work how they see best. 

A lot of companies, big and small, have adopted this approach. GitLab and Automattic for starters, but also [Twist](https://async.twist.com/asynchronous-communication/), [Buffer](https://buffer.com/about), and [Zapier](https://zapier.com/jobs/culture-and-values-at-zapier) have all followed a similar blueprint to make remote work actually work for both managers and engineers.
```

> ### The problem: Everyone will slack off

Reviewer 2: ```suggestion
### Problem #4: Lack of trust
```

I think this is a bit clearer. The problem isn't that people do slack off, it's that managers think they will. Will have to reword the below to fit a bit.

Study after study has also shown that remote workers are happier, more diverse, and that they stay at companies for longer and, as a result, do better work. That’s especially true [for engineers and developers](https://www.shakebugs.com/blog/why-let-developers-work-remotely/). Stackoverflow has a great guide that navigates the confusing [mess of statistics](https://stackoverflow.blog/2023/11/27/are-remote-workers-more-productive-that-s-the-wrong-question/) around productivity and remote work.

Ultimately, companies that want to make remote working work – and create a motivated, ambitious, and productive working environment – have to put a huge amount of trust in their employees. So, perhaps counter-intuitively, the solution to the perceived problem of productivity amongst remote workers is to go all-in on trust.

### The solution: Trust like you’ve never trusted before

> What does that mean in practice? First up, employers need to set clear and transparent expectations and then, within that framework, give people total freedom to get the job done. This kind of framework is great for engineers TKTK. Buffer co-founder and CEO Joel Gascoigne laid all this out [almost a decade ago](https://buffer.com/resources/why-transparency/). It’s still true today.

Reviewer 2: As per earlier comments, I'd like to see more examples of how real companies put this into practice + some examples of benefits if they're convincing. Just build the whole section around examples, rather than talking generally about the benefits of doing it.

Also "First up, employers need to set clear and transparent expectations and then, within that framework, give people total freedom to get the job done." just read like management speak. In general, we try to avoid the word "framework" where possible. I think it's a red flag for engineers.

Author: ```suggestion
What does that mean in practice? First up, employers need to set clear and transparent expectations and then, within that framework, give people total freedom to get the job done. Buffer co-founder and CEO Joel Gascoigne laid all this out [almost a decade ago](https://buffer.com/resources/why-transparency/). It’s still true today.
```

Author: Whoops, that was a placeholder. And if you've got any examples of some other companies/things companies are doing that would be a good fit here that would be really useful!

### The problem? Communication is messy

Making remote work actually work requires making big cultural and procedural changes. Fixing how your company communicates is one of the hardest as it requires everyone to buy in and change their behavior. That means saying goodbye to email, setting fire to Microsoft Teams, and training people to be much more effective communicators. Here’s how.

### The solution: Write. Everything. Down

> How do I [file expenses](https://posthog.com/handbook/people/spending-money)? What’s the [onboarding process for new hires](https://posthog.com/handbook/people/onboarding)? What’s the product analytics team [focused on this quarter](https://posthog.com/teams/product-analytics)? 
> 
> In most companies, answers to these kinds of questions are either hidden out-of-date Google Docs or inside someone’s brain. At well-functioning, all-remote companies, they’re written down in a handbook where anyone can access them. That handbook can be your public website, or an internal wiki. But written communication also includes Slack, GitHub, or systems like Trello or Airtable.
> 
> Defaulting to written communication means, chances are, if someone has a question they don’t have to wait for an answer – they can go find it themselves. And if they can’t find an answer, writing a clear, concise question in a public channel on Slack will ensure that everyone can see the answer. 
> 
> Being able to clearly communicate often complex ideas is a key skill for anyone working remotely, but especially engineers. Software engineer Dmitry Kudryavtsev [sums it up nicely](https://www.yieldcode.blog/post/why-engineers-should-write/): “The sooner you get better at writing, the faster you will become a better engineer.” It also created some lively discussion on [Hacker News](https://news.ycombinator.com/item?id=36659166).
> 
> Automattic’s Beau Lebens has written (of course) a [great guide](https://beau.blog/2020/03/remote-work-at-scale/) to the benefits of effective written communication with lots of practical tips. One key takeaway: tools are not the answer. Installing a bunch of bots to transcribe meetings and run automated functions in Slack will only work if you do all the hard work first. And nailing how you communicate is a big part of that. Writing things down is great because:
> 
> * **It’s faster. **Communicating via writing, especially with remote staff in lots of different time-zones, saves a lot of time. Scheduling is hard and meetings interrupt deep work. If something can be written down, it will both save time and improve transparency around decision-making.
> 
> * **It’s always there. **If a decision is made on a call and nobody writes it down, was a decision even made? Calls can be good, but calls are also information black holes. Transparency is crucial in remote-only companies. So, when you do have a meeting about something, make sure someone, [or something](https://www.sembly.ai/), is taking notes.
> 
> * **It’s clearer. **Seriously, have you ever read a transcript of you speaking? Writing things down helps with intentionality as, hopefully, it makes you think through both what you’re trying to achieve and how you want to communicate it.

Reviewer 2: ```suggestion
How do I [file expenses](https://posthog.com/handbook/people/spending-money)? Who is [our ideal customer](https://posthog.com/handbook/who-we-are-building-for)? What’s the product analytics team [focused on this quarter](https://posthog.com/teams/product-analytics)?

Don't hide the answers to these questions in out-of-date Google Docs, or worse still, inside someone’s brain. Instead, write them down in a handbook where anyone can access them. Ours is public, but an internal wiki is fine too.

Why? Because:

1. **It’s faster.** Scheduling is hard and meetings interrupt deep work. If something can be written down, it will save time and improve transparency around decision-making. Plus, it's more efficient to have a meeting to align everyone _after_ you've all written down what you think.

2. **It’s always there.** If a decision is made on a call and nobody writes it down, was a decision even made? Calls are information black holes so, when you do have a meeting about something, make sure someone, [or something](https://www.sembly.ai/), is taking notes.

3. **It’s clearer.** Seriously, have you ever read a transcript of you speaking? Writing things down makes you think through both what you’re trying to achieve and how you want to communicate it, making everyone more intentional.

Defaulting to written communication means, chances are, if someone has a question they don’t have to wait for an answer – they can go find it themselves. And if they can’t find an answer, writing a clear, concise question in a public channel on Slack will ensure that everyone can see the answer. 

Automattic’s Beau Lebens has [some great tips](https://beau.blog/2020/03/remote-work-at-scale/) on putting this into practice:

- **Tools are not the answer.** Installing a bunch of bots to transcribe meetings and run automated functions in Slack will only work if you do all the hard work first. And nailing how you communicate is a big part of that.

- **Another tip / warning**

- **Another tip / warning**
```

I was thinking something like this. I appreciate a second set of bullet points isn't ideal, but I think it's better to end on practical tips (do this/don't do that) than benefits.

Feel free to mess around with this, I just figured it was easier to lay it out than write a comment!

Reviewer 2: Edit: I removed this link. Maybe add this to the good reads at the end instead? https://www.yieldcode.blog/post/why-engineers-should-write/

> title: The best habits of the best remote teams

Reviewer 2: ```suggestion
title: The habits if effective remote teams
```

Had best twice

Author: I still think 'The habits of effective remote teams' lacks a certain something but I am struggling to think of an alternative.

How the best/most effective remote teams operate
Lessons from the most effective remote teams
What you can learn from the most effective remote teams

Reviewer 3: How the best remote teams actually work (/operate)

Author: quite like

How the best remote teams actually work

don't feel 'habits' is a good word for display copy

Reviewer 2: I've put them both in the poll. Can pick the winner.

date: 2024-05-09
author:
  - james-temperton
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/v1711656184/posthog.com/contents/images/newsletter/remote-work/remote_hog.png
featuredImageType: full
tags:
  - Culture
crosspost:
  - Founders
  - Blog
---

> Remote work sucks. If you work at Dell, that is. The company [recently announced](https://www.businessinsider.com/dell-remote-workers-promotion-return-office-push-flexible-work-2024-3) that remote employees would not be considered for promotions or be able to change roles. Dell’s rationale, which has something to do with “in-person connections” and “value differentiation,” is as tired as an entry-level Dell Inspiron trying to run _Cyberpunk 2077_.

Reviewer: I think this looks better as a start
```suggestion
Remote work sucks. If you work at Dell, that is. 

The company [recently announced](https://www.businessinsider.com/dell-remote-workers-promotion-return-office-push-flexible-work-2024-3) that remote employees would not be considered for promotions or be able to change roles. Dell’s rationale, which has something to do with “in-person connections” and “value differentiation,” is as tired as an entry-level Dell Inspiron trying to run _Cyberpunk 2077_.
```

Reviewer 2: +1

> It doesn’t have to be this way. The best remote teams are brimming with energy, they’re super productive, and do incredible work. So whether you’re part of a remote team, leading one, or want to go remote, here’s how the best of the best make it work, with some tips on how you can up your game, too.

Author: looks good!

### 1. They write everything down

Written communication is essential for successful remote teams. Documenting by default – be it information on how to [file expenses](https://posthog.com/handbook/people/spending-money), your [ICP](https://posthog.com/handbook/who-we-are-building-for), or what a specific team is [working on this quarter](https://posthog.com/teams/product-analytics) – gives everyone access to the same information, regardless of seniority or location.

Why is good, written communication essential at remote companies?

1. **It’s faster.** Scheduling is hard and meetings interrupt deep work. If something can be written down, it will save time and improve transparency around decision-making. Plus, it's more efficient to have a meeting to align everyone after you've all written down what you think.

> 2. **It’s always there.** If a decision is made on a call and nobody writes it down, was a decision even made? Calls are information black holes so, when you do have a meeting about something, make sure someone, [or something](https://www.sembly.ai), is taking notes.

Reviewer: Why this link?

Reviewer 2: +1 can remove this

At **Automattic**, Beau Lebens has [shared some great tips](https://beau.blog/2020/03/remote-work-at-scale/ on putting this into practice –) on how to communicate effectively as a remote team and explains traps to avoid. One key takeaway: tools are not the answer. Installing a bunch of bots to transcribe meetings and run automated functions in Slack will only help if you do all the hard work first.

At **Doist**, and just about any successful remote company, internal email is pretty much banned. As Doist founder and CEO Amir Salihefendic [explains](https://async.twist.com/asynchronous-communication/), email locks information away. This makes referring back to decisions or conversations either confusing or impossible. Default to communicating out in the open – GitHub, Slack, an internal wiki – so anyone can refer back to conversations and decisions at a later date.

### 2. They treasure deep work

> In an office setting, the [maker’s schedule](https://www.paulgraham.com/makersschedule.html) and the manager’s schedule (sort of) work side by side. But way too many teams switching to remote work have inadvertently defaulted to the manager’s schedule. On successful remote teams the manager’s schedule basically doesn’t exist, which makes them incredibly good at getting shit done.

Reviewer: ```suggestion
In an office setting, the [maker’s schedule](https://www.paulgraham.com/makersschedule.html) and the manager’s schedule (sort of) work side by side. But too many teams switching to remote work have inadvertently defaulted to the manager’s schedule. On successful remote teams the manager’s schedule basically doesn’t exist, which makes them incredibly good at getting shit done.
```

At **PostHog**, we’re [anti-meeting by default](https://posthog.com/blog/meetings). We have two meeting-free days a week and, when we do have meetings, they’re short, focused, and we take and share written notes where possible. We also only invite people who absolutely need to be there. We optimize for productivity, not presence.

At **Doist**, four days of each week are [dedicated to deep work](https://async.twist.com/heroes-housekeeping-days/), with Fridays reserved for housekeeping – bug fixes, battling technical debt, reading, research, and other smaller tasks. It’s an extreme version of time blocking – essentially blocking out four days a week for shipping, with one day reserved for everything else.

> When you absolutely have to have meetings, make the most of them. **GitLab** has a set of principles for how notes should be taken in [just about any meeting](https://handbook.gitlab.com/handbook/communication/#smart-note-taking-in-meetings). This starts with the agenda, which is shared in advance. During meetings, everyone is encouraged to help out with note-taking with a focus on capturing key points rather than every single word that’s said. 

Author: Maybe 'They prioritize deep work'. Point being these companies put in place a culture/team structure that gives people lots of time to focus on their important work. I think that comes across? But maybe 'treasure' isn't the right word?

> When you absolutely have to have meetings, make the most of them. **GitLab** has a set of principles for how notes should be taken in [just about any meeting](https://handbook.gitlab.com/handbook/communication/#smart-note-taking-in-meetings). This starts with the agenda, which is shared in advance. During meetings, everyone is encouraged to help out with note-taking with a focus on capturing key points rather than every single word that’s said. 

Reviewer 2: Yeah, 'prirotize' is better.

### 3. They value transparency and trust over process

The biggest (and stupidest) argument against remote work? It makes people less productive. Statistics on this issue abound, but here are some key findings: remote work [doesn’t affect productivity, at all](https://www.bloomberg.com/news/articles/2024-01-16/remote-work-doesn-t-seem-to-affect-productivity-fed-study-finds). That’s not according to some random survey, that’s based on detailed research by the Federal Reserve Bank of San Francisco. 

> Study after study has also shown that remote workers are happier, more diverse, and that they stay at companies for longer and, as a result, do better work. That’s especially true for [engineers and developers](https://www.shakebugs.com/blog/why-let-developers-work-remotely/). Stackoverflow has a great guide that navigates the [confusing mess of statistics](https://stackoverflow.blog/2023/11/27/are-remote-workers-more-productive-that-s-the-wrong-question/) around productivity and remote work.

Author: I think if the study we were quoting was some random piece of small fry academic research... yes. But that it's a major US institution...? I'm not wedded to it though, so we can cut. And for the graphic, I quite like the Gartner Hype Nonsense if we think the reference/gag works?

![Screenshot 2024-04-12 at 4 22 53 PM](https://github.com/PostHog/posthog.com/assets/162187331/d25f3222-30b0-4745-a110-4d62078b20a5)

> Study after study has also shown that remote workers are happier, more diverse, and that they stay at companies for longer and, as a result, do better work. That’s especially true for [engineers and developers](https://www.shakebugs.com/blog/why-let-developers-work-remotely/). Stackoverflow has a great guide that navigates the [confusing mess of statistics](https://stackoverflow.blog/2023/11/27/are-remote-workers-more-productive-that-s-the-wrong-question/) around productivity and remote work.

Reviewer 2: Yeah, I like this as well, but could you make it "Smart companies adapt". I don't think we're advocating for all companies everywhere closing their offices.

> Study after study has also shown that remote workers are happier, more diverse, and that they stay at companies for longer and, as a result, do better work. That’s especially true for [engineers and developers](https://www.shakebugs.com/blog/why-let-developers-work-remotely/). Stackoverflow has a great guide that navigates the [confusing mess of statistics](https://stackoverflow.blog/2023/11/27/are-remote-workers-more-productive-that-s-the-wrong-question/) around productivity and remote work.

Reviewer 2: RE: these two paras, I think it's just jarring that these two paras talk more about productivity than trust. I think a tweak along these lines would work:

- Managers believe remote workers aren't as productive.
- This is wrong. Here's some hard evidence and some examples (e.g. huge companies that are remote)
- The real problem? A low trust culture.

> Ultimately, teams and companies that really make remote working work have to go big on transparency and trust. The best asynchronous working environments are transparent by design, but how do you build trust? Here’s a checklist.

Author: ```suggestion
Ultimately, teams and companies that really make remote working work have to go big on transparency and trust. The best remote companies are transparent by design, but how do you build trust? Here’s a checklist.
```

- Give people autonomy. At **PostHog**, we hire people who work well with high level direction and who are comfortable stepping on toes to get things done. We trust people to do their best work and, in return, they deliver.

- Feedback. **GitLab** has a [policy](https://handbook.gitlab.com/handbook/values/) of “blameless problem solving.” Simply put, when something goes wrong, the investigation of the root cause is, as the company puts it, blameless. This encourages everyone to speak up without fear of punishment. People are trusted to make mistakes and, as a result, problems that do arise are easier to solve.

- Work in public. At **Zapier**, trust and transparency are a big part of the [company’s values](https://zapier.com/jobs/culture-and-values-at-zapier). As co-founder and CEO Wade Foster explains: “Transparency promotes equity and inclusion within our team and customer base. So work in public. Have discussions in the open and document your work.” The most successful remote companies build everything out in the open. This gives everyone the context they need to do their very best work across multiple time zones.

> - Shared values. The most successful remote teams have a shared set of values and behaviors – and these are often clearly defined in a public handbook. These values and behaviors are more important than dictatorial processes. As **Buffer** co-founder and CEO Joel Gascoigne wrote [almost a decade ago](https://buffer.com/resources/why-transparency/), “transparency breeds trust, and trust is is the foundation of great teamwork.” It’s still true today.

Author: Isn't that sort of covered throughout the newsletter? As in, a big chunk of those values are what we've walked people through already. I can repeat for emphasis, so something like...

The most successful remote teams have a shared set of values and behaviors – trust, transparency, autonomy – and these are often clearly defined in a public handbook.

> - Shared values. The most successful remote teams have a shared set of values and behaviors – and these are often clearly defined in a public handbook. These values and behaviors are more important than dictatorial processes. As **Buffer** co-founder and CEO Joel Gascoigne wrote [almost a decade ago](https://buffer.com/resources/why-transparency/), “transparency breeds trust, and trust is is the foundation of great teamwork.” It’s still true today.

Reviewer 2: Fair. In my head, I think I'm really talking about behaviors (e.g. Bias for impact etc.). Can leave this as is.

At **PostHog**, we organize an annual offsite. We’re far from alone, but we think we’re pretty good at them. The typical agenda includes a couple of big social events, team dinners, a day-long hackathon, all-hands strategy sessions and workshops, company culture exercises, and time for people to get some downtime and explore. 

Want to know how we plan great offsites? Look no further than [our handbook](https://posthog.com/handbook/company/offsites) – it even has a comprehensive checklist and some templates you can use to get started. But, for successful remote teams, meeting up goes beyond annual offsites. We also encourage individual teams to get together once a year to dig into specific projects and do in-person onboarding for all our new hires. Everyone at the company also has a quarterly travel budget so they can meet up with colleagues all over the world and solve some of the company’s biggest challenges.

> 

Reviewer: I do also think that budget for coworking really helps.

Reviewer 2: Would be good to have some stuff from other companies here, too.

Author: Was getting worried about length. I can link out to/namecheck other companies that go big on meet-ups/offsites, etc. but wouldn't want to add too much material. What do you reckon? And I'll add a line in about coworking budget for sure.

At **Doist**, four days of each week are [dedicated to deep work](https://async.twist.com/heroes-housekeeping-days/), with Fridays reserved for housekeeping – bug fixes, battling technical debt, reading, research, and other smaller tasks. It’s an extreme version of time blocking – essentially blocking out four days a week for shipping, with one day reserved for everything else.

When you absolutely have to have meetings, make the most of them. **GitLab** has a set of principles for how notes should be taken in [just about any meeting](https://handbook.gitlab.com/handbook/communication/#smart-note-taking-in-meetings). This starts with the agenda, which is shared in advance. During meetings, everyone is encouraged to help out with note-taking with a focus on capturing key points rather than every single word that’s said. 

### 3. They value transparency and trust over process

> ![The Remote Work Hype Cycle.]([https://myoctocat.com/assets/images/base-octocat.svg](https://res.cloudinary.com/dmukukwp6/image/upload/v1713294203/posthog.com/contents/images/newsletter/remote-work/hype_cycle.png))

Author: ```suggestion
![The Remote Work Hype Cycle](https://res.cloudinary.com/dmukukwp6/image/upload/v1713294203/posthog.com/contents/images/newsletter/remote-work/hype_cycle.png)

![James](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/posthog-first-five/james.png)
```

tags:
  - Culture
crosspost:
  - Founders
  - Blog
---

Remote work sucks. If you work at Dell, that is. The company [recently announced](https://www.businessinsider.com/dell-remote-workers-promotion-return-office-push-flexible-work-2024-3) that remote employees would not be considered for promotions or be able to change roles. Dell’s rationale, something to do with “in-person connections” and “value differentiation,” is as tired as an entry-level Dell Inspiron trying to run _Cyberpunk 2077_.

But the policy actually makes sense for Dell. Sort of. When you’re a decades-old behemoth built around people working in offices, implementing remote work policies that actually work is hard. But it doesn't have to be. There are some key principles that any company – big or small, young or old, or Dell – can follow to realize the potential of remote working. 

**This week, we explain: **how to make remote work actually work.

> This article was first published in our newsletter, [Product for Engineers](https://newsletter.posthog.com/). It's all about helping engineers and founders build better products by learning product skills. [Subscribe here](https://newsletter.posthog.com/subscribe).

> ### The problem? You’ve got a two-tiered system

Reviewer: Should all be h2
```suggestion
## The problem? You’ve got a two-tiered system
```

If you’re a company built around [big](https://www.cnbc.com/2023/06/08/google-to-crack-down-on-hybrid-work-asks-remote-workers-to-reconsider.html), [shiny](https://gizmodo.com/apple-remote-work-return-to-office-hybrid-work-1850256210) [offices](https://www.cnn.com/2022/06/01/tech/elon-musk-tesla-ends-work-from-home/index.html), then chances are you have a two-tiered system. Essentially, an employee’s ability to move up in such a company is closely linked to their proximity to the biggest, shiniest office. Dell’s new policy acknowledges this outright: if you’re a remote worker at Dell, there’s no career progression. The same is true at many companies that have a mix of remote, hybrid, and office workers.

### The solution? Aggressive transparency

Rather than presentism and nepotism, career progression should be linked to a transparent framework of measurable goals. Do the work, do it well, deliver value for customers, and you’ll be rewarded. Leif Singer, a software engineer at Automattic, the company behind WordPress, has shared some great examples of [how this works in practice](https://youtu.be/aRiP72kRrnc?feature=shared). 

> A two-tiered system is built on the bizarre notion that presenteeism is how you get ahead. Clearly defining and then publishing your system for career progression is an important first step to making sure that everyone, regardless of location, is assessed using the same objective criteria. At PostHog, the way we approach [career progression](https://posthog.com/handbook/people/career-progression) and [feedback](https://posthog.com/handbook/people/feedback) is laid out in public for anyone to see.

Reviewer: Once this is moved to Substack, you can remove `https://posthog.com` from all the links so that they are relative. 
```suggestion
A two-tiered system is built on the bizarre notion that presenteeism is how you get ahead. Clearly defining and then publishing your system for career progression is an important first step to making sure that everyone, regardless of location, is assessed using the same objective criteria. At PostHog, the way we approach [career progression](https://posthog.com/handbook/people/career-progression) and [feedback](https://posthog.com/handbook/people/feedback) is laid out in public for anyone to see.
```

Defaulting to written communication means, chances are, if someone has a question they don’t have to wait for an answer – they can go find it themselves. And if they can’t find an answer, writing a clear, concise question in a public channel on Slack will ensure that everyone can see the answer. Being able to clearly communicate often complex ideas is a key skill for anyone working remotely, but especially engineers. Software engineer Dmitry Kudryavtsev [sums it up nicely](https://www.yieldcode.blog/post/why-engineers-should-write/): “The sooner you get better at writing, the faster you will become a better engineer.” It also created some lively discussion on [Hacker News](https://news.ycombinator.com/item?id=36659166).

Automattic’s Beau Lebens has written (of course) a [great guide](https://beau.blog/2020/03/remote-work-at-scale/) to the benefits of effective written communication with lots of practical tips. One key takeaway: tools are not the answer. Installing a bunch of bots to transcribe meetings and run automated functions in Slack will only work if you do all the hard work first. And nailing how you communicate is a big part of that. Writing things down is great because:

> * **It’s faster. **Communicating via writing, especially with remote staff in lots of different time-zones, saves a lot of time. Scheduling is hard and meetings interrupt deep work. If something can be written down, it will both save time and improve transparency around decision-making.
> * **It’s always there. **If a decision is made on a call and nobody writes it down, was a decision even made? Calls can be good, but calls are also information black holes. Transparency is crucial in remote-only companies. So, when you do have a meeting about something, make sure someone, [or something](https://www.sembly.ai/), is taking notes.
> * **It’s clearer. **Seriously, have you ever read a transcript of you speaking? Writing things down helps with intentionality as, hopefully, it makes you think through both what you’re trying to achieve and how you want to communicate it.

Reviewer: For long list items, we add a break between them to make them more readable.
```suggestion
* **It’s faster. **Communicating via writing, especially with remote staff in lots of different time-zones, saves a lot of time. Scheduling is hard and meetings interrupt deep work. If something can be written down, it will both save time and improve transparency around decision-making.

* **It’s always there. **If a decision is made on a call and nobody writes it down, was a decision even made? Calls can be good, but calls are also information black holes. Transparency is crucial in remote-only companies. So, when you do have a meeting about something, make sure someone, [or something](https://www.sembly.ai/), is taking notes.

* **It’s clearer. **Seriously, have you ever read a transcript of you speaking? Writing things down helps with intentionality as, hopefully, it makes you think through both what you’re trying to achieve and how you want to communicate it.
```

### The problem: Meetings

Online meetings are the scourge of remote work – and often a cause of tension between office-based and non-office based employees. Mics off when they should be on, screenshare chaos, [Jeffrey Toobin’s penis](https://www.wired.com/story/jeffrey-toobin-zoom-penis/) – they remain a technical nightmare and a productivity black hole. Many companies see them as a necessary evil – a way for colleagues, no matter where they’re based, to collaborate. This is wrong.

> Since the pandemic, the amount of time people have spent not just in online meetings, but in meetings in general, has soared. [Time spent in meetings](https://www.microsoft.com/en-us/worklab/work-trend-index/great-expectations-making-hybrid-work-work?ref=async.twist.com) has gone up 253%. Some Teams users [spend](https://www.microsoft.com/en-us/worklab/work-trend-index/will-ai-fix-work) close to eight hours a week, or an entire workday, in online meetings. Eight hours a week! In Teams! Madness. And if you’re a manager or executive within an organization, this is all your fault.

Reviewer: ```suggestion
Since the pandemic, the amount of time people have spent not just in online meetings, but in meetings in general, has soared. [Time spent in meetings](https://www.microsoft.com/en-us/worklab/work-trend-index/great-expectations-making-hybrid-work-work) has gone up 253%. Some Teams users [spend](https://www.microsoft.com/en-us/worklab/work-trend-index/will-ai-fix-work) close to eight hours a week, or an entire workday, in online meetings. Eight hours a week! In Teams! Madness. And if you’re a manager or executive within an organization, this is all your fault.
```

Focus on output, not time-in-seat. As an employer, you need to be open and honest and demand the same of your employees. Building an organization around trust also means you need to hire carefully. Give the wrong people too much trust and, yes, they might abuse it. But give the right people trust and you’ll create a company that thrives on autonomy. 

Companies that have made remote working work also have one other thing in common: transparency. It’s super important. PostHog co-founder James Hawkins [explains why](https://posthog.com/founders/how-to-run-a-transparent-company): “Transparency makes the whole company a joint project, between our team and the community around it too.”

That might sound a bit effusive, but getting it right has a tangible impact. Transparency about how you run a company helps employees buy into what you’re building. Here’s why trust and transparency are key for remote work:

> * **Empowerment: **Hire the right people and give them the freedom to do their best work. This creates a culture of ownership which, in turn, makes people more engaged and productive.
> * **Openness: **Be aggressively transparent and default to communicating out in the open. This not only makes asynchronous working easier, it also helps bring remote teams closer together.
> * **Investment: **Trust gets you a long way, but you also have to invest to really make that trust work. Almost all successful remote companies do offsites once a year. At PostHog, we also provide all employees with a [quarterly travel budget](https://posthog.com/handbook/people/spending-money) to meet up with colleagues for work and fun.

Reviewer: ```suggestion
* **Empowerment: **Hire the right people and give them the freedom to do their best work. This creates a culture of ownership which, in turn, makes people more engaged and productive.

* **Openness: **Be aggressively transparent and default to communicating out in the open. This not only makes asynchronous working easier, it also helps bring remote teams closer together.

* **Investment: **Trust gets you a long way, but you also have to invest to really make that trust work. Almost all successful remote companies do offsites once a year. At PostHog, we also provide all employees with a [quarterly travel budget](https://posthog.com/handbook/people/spending-money) to meet up with colleagues for work and fun.
```

> title: The habits of effective remote teams

Reviewer: ```suggestion
---
title: The habits of effective remote teams
```

date: 2024-05-09
author:
  - james-temperton
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/v1711656184/posthog.com/contents/images/newsletter/remote-work/remote_hog.png
featuredImageType: full
tags:
  - Culture
crosspost:
  - Founders
  - Blog
---

Remote work sucks. If you work at Dell, that is.

In March, it [announced](https://www.businessinsider.com/dell-remote-workers-promotion-return-office-push-flexible-work-2024-3) remote employees would not be considered for promotions, or be able to change roles. Dell’s rationale, some waffle about “in-person connections” and “value differentiation,” is as tired as a Dell Inspiron trying to run _Cyberpunk 2077_.

It doesn’t have to be this way. The best remote teams are brimming with energy, are super productive, and do incredible work. So whether you’re part of a remote team, leading one, or want to go remote, here’s how the best of the best make it work, with some tips on how you can up your game, too.

> ### **1. They write everything down**

Reviewer: ```suggestion
### 1. They write everything down
```

Remote work doesn't work without a strong writing culture.

> Documenting by default – be it information on how to [file expenses](https://posthog.com/handbook/people/spending-money), your [ICP](https://posthog.com/handbook/who-we-are-building-for), or what a specific team is [working on this quarter](https://posthog.com/teams/product-analytics) – gives everyone access to the same information, regardless of seniority or location.

Reviewer: ```suggestion
Documenting by default – be it information on how to [file expenses](/handbook/people/spending-money), your [ICP](/handbook/who-we-are-building-for), or what a specific team is [working on this quarter](/teams/product-analytics) – gives everyone access to the same information, regardless of seniority or location.
```

Why does this work? Because:

1. **It’s faster.** Scheduling is hard and meetings interrupt deep work. If something can be written down, it will save time and improve transparency around decision-making. Plus, it's more efficient to have a meeting to align everyone after you've all written down what you think.

2. **It’s always there.** If a decision is made on a call and nobody writes it down, was a decision even made? Calls are information black holes so, when you do have a meeting about something, make sure someone, or something, is taking notes. Better yet, don't have a meeting.

3. **It’s clearer.** Seriously, have you ever read a transcript of you speaking? Writing is thinking, so it helps you think through both what you’re trying to achieve and how you want to communicate it, making everyone more intentional.

How does this work in practice?

> At **PostHog**, it started with [our company handbook](https://posthog.com/handbook) – it’s one of the most important things we’ve ever made. It helps everyone understand what we're trying to build and why. Handbooks don't have to be public, but we were inspired by GitLab's [2,000-page handbook](https://handbook.gitlab.com/handbook/values/), which covers every aspect of how its 2,000+ team members work across 65+ countries.

Reviewer: ```suggestion
At **PostHog**, it started with [our company handbook](/handbook) – it’s one of the most important things we’ve ever made. It helps everyone understand what we're trying to build and why. Handbooks don't have to be public, but we were inspired by GitLab's [2,000-page handbook](https://handbook.gitlab.com/handbook/values/), which covers every aspect of how its 2,000+ team members work across 65+ countries.
```

At **Doist**, and just about any successful remote company, internal email is pretty much banned. As Doist founder and CEO Amir Salihefendic [explains](https://async.twist.com/asynchronous-communication/), email locks information away. This makes referring back to decisions or conversations either confusing or impossible. Default to communicating out in the open – GitHub, Slack, an internal wiki – so anyone can find and reference conversations and decisions at a later date.

![Hype cycle](https://res.cloudinary.com/dmukukwp6/image/upload/v1715266314/posthog.com/contents/blog/hype-cycle-1.png)

### 2. They prioritize deep work

In an office setting, the [maker’s schedule](https://www.paulgraham.com/makersschedule.html) and the manager’s schedule (sort of) work side by side. But the manager's schedule is the enemy of remote work. Successful remote teams avoid it entirely, which makes them incredibly good at getting shit done.

> At **PostHog**, we’re [anti-meeting by default](https://posthog.com/blog/meetings). We have two meeting-free days a week and, when we do have meetings, they’re short, focused, and we take and share written notes where possible. We also only invite people who absolutely need to be there. We optimize for productivity, not presence.

Reviewer: ```suggestion
At **PostHog**, we’re [anti-meeting by default](/blog/meetings). We have two meeting-free days a week and, when we do have meetings, they’re short, focused, and we take and share written notes where possible. We also only invite people who absolutely need to be there. We optimize for productivity, not presence.
```

At **Doist**, four days of each week are [dedicated to deep work](https://async.twist.com/heroes-housekeeping-days/), with Fridays reserved for housekeeping – bug fixes, battling technical debt, reading, research, and other smaller tasks. It’s an extreme version of time blocking – essentially blocking out four days a week for shipping, with one day reserved for everything else.

When you absolutely have to have meetings, make the most of them. **GitLab** has a set of principles for how notes should be taken in [just about any meeting](https://handbook.gitlab.com/handbook/communication/#smart-note-taking-in-meetings). This starts with the agenda, which is shared in advance. During meetings, everyone is encouraged to help out with note-taking with a focus on capturing key points rather than every single word that’s said.

> > This post was first published in our newsletter, [_Product for Engineers_](https://newsletter.posthog.com/). It's all about helping engineers and founders build better products by learning product skills. Subscribe [here]([url](https://newsletter.posthog.com/)).

Reviewer: ```suggestion
> This post was first published in our newsletter, [_Product for Engineers_](https://newsletter.posthog.com/). It's all about helping engineers and founders build better products by learning product skills. Subscribe [here](https://newsletter.posthog.com/).
```

* **Shared values.** The most successful remote teams have a shared set of values and behaviors – and these are often clearly defined in a public handbook. These values and behaviors are more important than dictatorial processes. As Buffer co-founder and CEO Joel Gascoigne wrote [almost a decade ago](https://buffer.com/resources/why-transparency/), “transparency breeds trust, and trust is is the foundation of great teamwork.” It’s still true today.

### 4. They make meeting up easy

Pretty much all successful remote teams have one thing in common: they’re not remote all the time.

That doesn’t mean hybrid working, or mandatory office days, it means both encouraging meeting up and making it easy to do so.

At **PostHog**, we achieve this in a bunch of different ways:

* **$300 monthly co-working budget**.** This makes it easy for people to meet up and work together if they already live in the same city.

> * **$1,500 quarterly meet-up budget** for ad hoc in-person collaboration with colleagues. This can cover flights, accommodation, and other expenses. At PostHog, Paul and Ben used this recently to meet for a week in Italy and ship our new [heatmap and scroll depth feature](https://posthog.com/changelog/2024#mouse-and-scroll-depth-tracking-launched).

Reviewer: ```suggestion
* **$1,500 quarterly meet-up budget** for ad hoc in-person collaboration with colleagues. This can cover flights, accommodation, and other expenses. At PostHog, Paul and Ben used this recently to meet for a week in Italy and ship our new [heatmap and scroll depth feature](/changelog/2024#mouse-and-scroll-depth-tracking-launched).
```

* **In-person onboarding.** When new people join the company, we prioritize organizing an in-person onboarding session as soon as possible. This helps people get to know each other and share important context.

* **Annual company offsites.** The typical agenda includes a couple of big social events, team dinners, a hackathon, all-hands strategy sessions, company culture exercises, and downtime to socialize and explore.

> * **Frequent small team offsites.** In addition to our all-company offsite, all our [small teams](https://posthog.com/handbook/team-structure) run at least one annual offsite to focus on specific team goals and intense, in-person collaboration. They often include a hackathon, too, but this varies.

Reviewer: ```suggestion
* **Frequent small team offsites.** In addition to our all-company offsite, all our [small teams](/handbook/team-structure) run at least one annual offsite to focus on specific team goals and intense, in-person collaboration. They often include a hackathon, too, but this varies.
```

Our goal with all of this is to make our in-person time as intentional as possible. To work on big, specific problems, build bonds that create trust, and make asynchronous work more creative and collaborative.

### Good reads 📖

> **[Building an all-remote company from scratch](https://posthog.com/founders/remote-culture) – James Hawkins** This is exactly how we built a fully-remote company from the ground up. This was written in 2020 when we were 12 people. Today, we’re almost 50 and it’s still working.

Reviewer: ```suggestion
**[Building an all-remote company from scratch](/founders/remote-culture) – James Hawkins** 
This is exactly how we built a fully-remote company from the ground up. This was written in 2020 when we were 12 people. Today, we’re almost 50 and it’s still working.
```

> **[The manager’s schedule is holding remote work back](https://marker.medium.com/the-managers-schedule-is-holding-remote-work-back-f9c1302ac6f3) – Luke Thomas** Many makers (hello, product engineers!) have been working remotely successfully for years, but managers can often struggle. Luke shares how to fix that problem.

Reviewer: ```suggestion
**[The manager’s schedule is holding remote work back](https://marker.medium.com/the-managers-schedule-is-holding-remote-work-back-f9c1302ac6f3) – Luke Thomas** 
Many makers (hello, product engineers!) have been working remotely successfully for years, but managers can often struggle. Luke shares how to fix that problem.
```

> **[Why remote work is so hard – and how it can be fixed](https://www.newyorker.com/culture/annals-of-inquiry/can-remote-work-be-fixed) – Cal Newport** This is a great, detailed read on the long-standing problems that have scuppered remote work. It’s written by Cal Newport, who also coined the term “deep work.”

Reviewer: ```suggestion
**[Why remote work is so hard – and how it can be fixed](https://www.newyorker.com/culture/annals-of-inquiry/can-remote-work-be-fixed) – Cal Newport** 
This is a great, detailed read on the long-standing problems that have scuppered remote work. It’s written by Cal Newport, who also coined the term “deep work.”
```
